### DISCUSSION TITLE
Co-STORM

### DISCUSSION THREAD

**Discussion by danieldekay** [assafelovic/gpt-researcher#816] [Ideas]
The Co-STORM paper describes how they are using human-in-the-loop and a multitude of expert agents, moderated by a moderator agent, to create better output.

https://arxiv.org/abs/2408.15232

**Comment by mdlmarkham** [comment]
Have you seen the implementation of STORM here: https://storm.genie.stanford.edu/ there is a GitHub repo - may be something useful there.

**Reply by danieldekay** [reply]
Yes, i have used STORM a bit. It’s quite good. But the open source code is not what you see on the demo: the front end is missing and the final writing step that includes the correct citations. 

And the Co-STORM has human in the loop which is not even available on their demo, let alone the GitHub repository.
